tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+            '<md-button class="md-primary" ng-click="dismiss()">{{cancel}}</md-button>' +
+            '<md-button class="md-primary" ng-click="submit()">{{ok}}</md-button>' +
+          '</div>' +
+        '</md-edit-dialog>'
+    };
+  }
+  
+  this.show = function (options) {
+    if(busy) {
+      return $q.reject();
+    }
+    
+    busy = true;
+    options = angular.extend({}, defaultOptions, options);
+    
+    if(!options.targetEvent) {
+      return logError('options.targetEvent is required to align the dialog with the table cell.');
+    }
+    
+    if(!options.targetEvent.currentTarget.classList.contains('md-cell')) {
+      return logError('The event target must be a table cell.');
+    }
+    
+    if(options.bindToController && !options.controllerAs) {
+      return logError('You must define options.controllerAs when options.bindToController is true.');
+    }
+    
+    options.target = options.targetEvent.currentTarget;
+    
+    var promise = getTemplate(options);
+    var promises = [promise];
+    
+    for(var prop in options.resolve) {
+      promise = options.resolve[prop];
+      promises.push($q.when(angular.isFunction(promise) ? promise() : promise));
+    }
+    
+    promise = $q.all(promises);
+    
+    promise['catch'](logError);
+    
+    return promise.then(function (results) {
+      var template = results.shift();
+      
+      for(var prop in options.resolve) {
+        options.resolve[prop] = results.shift();
+      }
+      
+      return build(template, options);
+    });
+  };
+  
+  this.small = function (options) {
+    return this.show(angular.extend({}, options, preset('small', options)));
+  }.bind(this);
+  
+  this.large = function (options) {
+    return this.show(angular.extend({}, options, preset('large', options)));
+  }.bind(this);
+  
+  return this;
+}
+
+mdEditDialog.$inject = ['$compile', '$controller', '$document', '$mdUtil', '$q', '$rootScope', '$templateCache', '$templateRequest', '$window'];
+
+
+angular.module('md.data.table').directive('mdFoot', mdFoot);
+
+function mdFoot() {
+
+  function compile(tElement) {
+    tElement.addClass('md-foot');
+  }
+
+  return {
+    compile: compile,
+    restrict: 'A'
+  };
+}
+
+angular.module('md.data.table').directive('mdHead', mdHead);
+
+function mdHead($compile) {
+
+  function compile(tElement) {
+    tElement.addClass('md-head');
+    return postLink;
+  }
+  
+  // empty controller to be bind scope properties to
+  function Controller() {
+    
+  }
+  
+  function postLink(scope, element, attrs, tableCtrl) {
+    // because scope.$watch is unpredictable
+    var oldValue = new Array(2);
+    
+    function addCheckboxColumn() {
+      element.children().prepend('<th class="md-column md-checkbox-column">');
+    }
+    
+    function attatchCheckbox() {
+      element.prop('lastElementChild').firstElementChild.appendChild($compile(createCheckBox())(scope)[0]);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933951)
```suggestion
      element.prop("lastElementChild").firstElementChild.appendChild($compile(createCheckBox())(scope)[0]);
```